internal/sdkprovider/service/kafkatopic/kafka_topic_cache.go

```diff
@@ -201,3 +201,11 @@ func DeleteTopicFromCache(projectName, serviceName, topicName string) {
 	}
 	t.Unlock()
 }
+
+// GetFullQueue retrieves a topics queue
+func (t *kafkaTopicCache) GetFullQueue(projectName, serviceName string) []string {
+	t.RLock()
+	defer t.RUnlock()
+
+	return t.inQueue[projectName+serviceName]
```

Comment:
consider handling a scenario where the key might not exist in the `inQueue` map; directly accessing a non-existent key will return a nil slice, which might be unexpected for the function caller